misc/custom.scss

```diff
@@ -353,16 +298,8 @@ table.table-h {
   margin-left: 1rem;
   margin-right: 1rem;
   padding: 2rem;
-  margin-top: -7.5rem;
+  margin-top: -5rem;
```

Comment:
the changes here (and below in this file) appears to be actual changes rather than auto format - may i get some context here on why these chnages were needed?

Reply:
or did they already exist on the `custom.scss` of the site we copied from and the diff is between `do-not-touch` and `custom.scss`

Reply:
The second! What we did here was just copy `custom.scss` that would exist on a fresh repo created by site creation, and replace the existing things in `custom.scss` - i think the styles changed slightly over time but we never updated our base template